includes/html/print-alert-templates.php

```diff
@@ -11,28 +11,38 @@
           <tr>
             <th data-column-id="id" data-searchable="false" data-identifier="true" data-type="numeric">#</th>
             <th data-column-id="templatename">Name</th>
+            <th data-column-id="alert_rules" data-searchable="false" data-formatter="alert_rules">Alert Rules</th>
             <th data-column-id="actions" data-searchable="false" data-formatter="commands">Action</th>
             <th data-column-id="old_template" data-searchable="false" data-visible="false">Old template</th>
           </tr>
       </thead>
       <tbody>
 <?php
-$full_query = dbFetchRows('SELECT id, name, template from alert_templates', $param);
+$full_query = dbFetchRows('SELECT id, name, template FROM alert_templates', $param);
 foreach ($full_query as $template) {
     if ($template['name'] == 'Default Alert Template') {
         $default_tplid = $template['id'];
         $template['id'] = 0;
+        $db_alert_rules = dbFetchRows('SELECT r.id, r.name FROM alert_rules AS r where r.id NOT IN (SELECT alert_rule_id FROM alert_template_map) ORDER BY r.name');
+    } else {
+        $db_alert_rules = dbFetchRows('SELECT r.id as id, r.name as name FROM alert_rules AS r LEFT JOIN alert_template_map AS m ON r.id=m.alert_rule_id WHERE m.alert_templates_id = ? ORDER BY r.name', $template['id']);
```

Comment:
This introduces `N+1` queries, cant you rewrite this to an eloquent query and use eager loading?

Reply:
Still the same issue. You should add relationship in the models and use that to eager load the data instead